__tests__/Unit/Components/ExtensionRequest/ExtensionRequestForm.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { ExtensionRequestForm } from '@/components/ExtensionRequest/ExtensionRequestForm';
+import { toast } from '@/helperFunctions/toast';
+
+const mockCreateExtensionRequest = jest.fn();
+
+jest.mock('@/app/services/tasksApi', () => ({
+    useCreateExtensionRequestMutation: () => [
+        mockCreateExtensionRequest,
+        { isLoading: false },
+    ],
+}));
+
+jest.mock('@/helperFunctions/toast', () => ({
+    toast: jest.fn(),
+    ToastTypes: {
+        SUCCESS: 'SUCCESS',
+        ERROR: 'ERROR',
+    },
+}));
+
+const defaultProps = {
+    isOpen: true,
+    onClose: jest.fn(),
+    taskId: '123',
+    assignee: 'john',
+    oldEndsOn: new Date().getTime(),
+};
+
+describe('ExtensionRequestForm Component', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('renders all fields correctly', () => {
```

Comment:
we use `should` as convention in tests
```suggestion
    test('should render all fields correctly', () => {
```

Reply:
done